lib/builder.js

```diff
@@ -1,4 +1,5 @@
 /** @babel */
+/* eslint-disable no-multi-spaces */
```

Comment:
What's the rationale behind this? Has something changed in JavaScript Standard Style recently?

Reply:
The linter is complaining about multiple spaces before comments. Without the override the columns won't line up, unless you want to override it for the whole project?

Reply:
Not worth keeping the alignment for this file. For the parser files that contain aligned EOL comments, we should disable the rule only for those blocks of regex patterns, and re-enable immediately after them.